networking_generic_switch/devices/corsa_devices/corsavfc.py

```diff
@@ -451,6 +447,555 @@ def bridge_detach_tunnel(headers,
     return output
 
 
+#
+# GET DATAPATH
+#
+#   200 OK
+#   303 See Other
+def get_datapath(headers,
+                 url_switch):
+    url = url_switch + ep_datapath
+
+    try:
+        r = requests.get(url, headers=headers, verify=False)
+    except Exception as e:
+        raise e
+    return r
+
+
+#
+# GET DATAPATH LAG-HASH
+#
+#   200 OK
+def get_datapath_laghash(headers,
+                         url_switch):
+    url = url_switch + ep_datapath + '/lag-hash'
+
+    try:
+        r = requests.get(url, headers=headers, verify=False)
+    except Exception as e:
+        raise e
+    return r
+
+
+#
+# GET DATAPATH STATUS
+#
+#   200 OK
+#   303 See other
+def get_datapath_status(headers,
+                        url_switch):
+    url = url_switch + ep_datapath + '/status'
+
+    try:
+        r = requests.get(url, headers=headers, verify=False)
+    except Exception as e:
+        raise e
+    return r
+
+
+#
+# GET EQUIPMENT
+#
+#   200 OK
+#
+def get_equipment(headers,
+                  url_switch):
+    url = url_switch + ep_equipment
+
+    try:
```

Comment:
General comment: if you are doing `except... raise`, there is no reason to have a `try` in the first place. The reason is that the exception that is caught would immediately be re-raised without being handled, just as if you did not attempt to catch it at all. Most of these functions then become one-liners:

```python
def get_equipment(headers, url_switch):
    return requests.get(url_switch + ep_equipment, headers=headers, verify=False)
```